tasks/licenses.py

```diff
@@ -196,7 +196,7 @@ def find_copyright_for(package, overrides, ctx):
 
     for filename in COPYRIGHT_LOCATIONS:
         filename = os.path.join(pkgdir, filename)
-        if os.path.exists(filename):
+        if os.path.exists(filename) and not os.path.isdir(filename):
```

Comment:
```suggestion
        if os.path.isfile(filename):
```
I think [this](https://docs.python.org/3/library/os.path.html#os.path.isfile) would be shorter, WDYT?

Reply:
Yes, I missed that function :+1: